### PR TITLE
transition to Swift 3.0

### PR DESCRIPTION
**Pull request by galblank**
Updated main swift file to support latest Swift

### CODE REVIEW

**Comment by rivera-ernesto** [comment]
Created a separate branch for this https://github.com/CocoaLumberjack/CocoaLumberjack/tree/swift_3.

File: Classes/CocoaLumberjack.swift [hunk 1]
```diff
@@ -19,77 +19,73 @@ extension DDLogFlag {
...
-                return .verbose
-            } else if contains(.debug) {
-                return .debug
-            } else if contains(.info) {
-                return .info
-            } else if contains(.warning) {
-                return .warning
-            } else if contains(.error) {
-                return .error
+            if contains(.Verbose) {
```
Comment (max-potapov) [inline Classes/CocoaLumberjack.swift]: It's not a Swift 3.0 syntax.

File: Classes/CocoaLumberjack.swift [hunk 2]
```diff
@@ -19,77 +19,73 @@ extension DDLogFlag {
...
-    _DDLogMessage(message, level: level, flag: .error, context: context, file: file, function: function, line: line, tag: tag, asynchronous: async, ddlog: ddlog)
+public func DDLogError(_ message: () -> String, level: DDLogLevel = defaultDebugLevel, context: Int = 0, file: StaticString = #file, function: StaticString = #function, line: UInt = #line, tag: AnyObject? = nil, asynchronous async: Bool = false, ddlog: DDLog = DDLog.sharedInstance()) {
+    _DDLogMessage(message, level: level, flag: DDLogFlag.Error, context: context, file: file, function: function, line: line, tag: tag, asynchronous: async, ddlog: ddlog)
 }
 
 /// Returns a String of the current filename, without full path or extension.
 ///
 /// Analogous to the C preprocessor macro `THIS_FILE`.
 public func CurrentFileName(_ fileName: StaticString = #file) -> String {
-    var str = String(describing: fileName)
```
Comment (max-potapov) [inline Classes/CocoaLumberjack.swift]: It seems you have converted Swift 3.0 to Swift 2.3.

Comment (bpoplauschi) [inline Classes/CocoaLumberjack.swift]: Agree with @max-potapov. I will close this PR.

**Pull request closed without merging**